src/CalcViewModel/Common/LocalizationService.h

```diff
@@ -39,6 +39,21 @@ namespace CalculatorApp { namespace Common
         Windows::UI::Text::FontWeight GetFontWeightOverride();
         double GetFontScaleFactorOverride(LanguageFontType fontType);
 
+        void Sort(std::vector<Platform::String^>& source);
```

Comment:
Unused?

Reply:
Yes, but it's probably better to provide a method to directly sorts strings without using one with a lambda (better performance). If it was an internal project, I would agree, but because it's an open-source project, it's probably better to provide some extra functions/helpers, especially in a class like `LocalizationService`, I see this class as a toolbox, it will probably help another developer in a future.